File: runtimes/node/actions/process-change.js
function main(params) {

  return new Promise(function(resolve, reject) {
    if (!params.name || !params.color) {
      reject({
        'error': 'Please make sure name and color are passed in as params.'
      });
      return;
    } else {
      var message = 'A ' + params.color + ' cat named ' + params.name + ' was added.';
      console.log(message);
      resolve({
        change: message
      });
      return;
    }
  });
}

File: runtimes/swift/actions/process-change.swift
func main(args: [String:Any]) -> [String:Any] {
    if let color = args["color"] as? String,
        let name = args["name"] as? String
    {
      let message = "A \(color) cat named \(name) was added."
      print(message)
      return [ "change": message ]
    } else {
      return [ "error": "Please make sure name and color are passed in as params." ]
    }
}
